docs/ref/models/database-functions.txt

```diff
@@ -733,7 +753,24 @@ It can also be registered as a transform. For example::
     >>> # Get authors whose name is longer than 7 characters
     >>> authors = Author.objects.filter(name__length__gt=7)
 
-``Lower``
+
+``Left``
+---------
+
+.. class:: Left(expression, length, **extra)
+
+Accepts a single text field or expression and the 1-indexed length of the substring to return.
```

Comment:
I think details about Oracle and MySQL should be mentioned here. Please also wrap text at 79 chars.